special-pages/pages/onboarding/app/components/v3/SettingsStep.js

```diff
@@ -73,6 +74,8 @@ export function SettingsStep({ data }) {
 export function SettingListItem({ index, item, dispatch }) {
     const data = item.data;
     const { t } = useTypedTranslation();
+    const { isDarkMode } = useEnv();
+    const platformName = /** @type {'macos'|'windows'} */ (usePlatformName());
```

Comment:
This is the big difference, now the *consumer* of the switch controls whether it’s light or dark